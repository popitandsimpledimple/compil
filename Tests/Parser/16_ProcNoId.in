procedure (var a: Double; const b: Integer; c: String);
begin end;
begin
end.