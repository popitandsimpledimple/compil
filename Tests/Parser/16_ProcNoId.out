(1,11) Expect Identifier